vignettes/UserGuide.Rmd

```diff
@@ -171,25 +172,26 @@ CP.default.tree <- create.phylogenetic.tree(
     );
 ```
 
-To control the polygon shape and size, users can modify the `polygon.shape` and `polygon.width` parameters. The `colour.scheme` parameter can be used to specify a colour palette for the polygons. When a single colour is specified, CEV will create a gradient from white to the specified colour and assign progressively darker shades to each node.
+To control the overall scale of the polygons, users can modify the `polygon.width` parameters. The `colour.scheme` parameter can be used to specify a colour palette for the polygons. When a single colour is specified, CEV will create a gradient from white to the specified colour and assign progressively darker shades to each node.
 
 ```{r, fig.height=4}
 CP.tree <- create.phylogenetic.tree(
     CP,
-    colour.scheme = 'steelblue',
-    polygon.shape = 2,
-    polygon.width = 2
+    polygon.colour.scheme = 'steelblue',
+    polygon.scale = 2
     );
 ```
 
 Alternatively, the `colour.scheme` parameter can take a vector with length equal to the number of nodes. This will assign a unique colour to each node.
 ```{r, fig.height=4}
 CP.tree2 <- create.phylogenetic.tree(
     CP,
-    colour.scheme = c(NA, 'gray70', 'pink', 'maroon', 'steelblue')
+    polygon.colour.scheme = c('gray70', 'pink', 'maroon', 'steelblue')
```

Comment:
Is this still supported? Does this mean assigning node-specific polygon colours can be done through this or the dataframe?
Maybe we should add the different ways users can assign polygon colours and how we recommend using them.

Reply:
True, I'll add to this section first and see if I can clear it up for you in the User Guide. Will be a good test of the docs.